aws_lambda_builders/workflows/nodejs_npm_esbuild/workflow.py

```diff
@@ -127,7 +127,11 @@ def actions_with_bundler(
         )
 
         install_action = NodejsNpmWorkflow.get_install_action(
-            source_dir, scratch_dir, subprocess_npm, osutils, self.options, is_production=False
+            source_dir,
+            scratch_dir,
+            subprocess_npm,
+            osutils,
+            self.options,
```

Comment:
I assume `is_production` defaults to `True`?

Is there any reason a customer may want `False`? If so, how would a customer do that?

Reply:
Yes, it defaults to true. We don't support any way for customers to set it to False, and we actually don't use it as False anywhere except here so I'm just going to remove that parameter. Setting `production=False` means that development dependencies will also be installed which adds a ton of bloat and install time that customers likely won't want packaged with their Lambda. Before, we expected customers to include `esbuild` as a dev dependency. Now, we expect it to be installed on their PATH or as a normal dependency. This is the behaviour we currently have with our Node.js workflow and are making esbuild consistent.